libnmstate/nm/sriov.py

```diff
@@ -72,8 +140,65 @@ def get_info(device):
     )
 
     if sriov_setting:
-        info[Ethernet.SRIOV_SUBTREE] = {
-            Ethernet.SRIOV.TOTAL_VFS: sriov_setting.props.total_vfs
-        }
+        sriov_config = {}
+        ifname = device.get_iface()
+        with open(f"/sys/class/net/{ifname}/device/sriov_numvfs", "r") as f:
+            sriov_config[Ethernet.SRIOV.TOTAL_VFS] = int(f.read())
```

Comment:
If possible, please split this change into another commit or PR.

Reply:
Done! I added the change in another commit